src/app/code/community/Meanbee/ConfigPoweredCss/Model/Config.php

```diff
@@ -77,6 +78,66 @@ public function getCssFilename($store = null)
      */
     public function getFullCssFilePath($store = null)
     {
-        return Mage::getBaseDir('skin') . '/frontend/base/default/' . $this->getCssFilename($store);
+        return $this->getCssDirectoryPath() . $this->getCssFilename($store);
+    }
+
+    /**
+     * Get directory for CSS files
+     *
+     * @param $store
+     * @return string
+     */
+    public function getCssDirectoryPath($store = null)
+    {
+
+        return Mage::getBaseDir('media') . $this->getCssPath($store);
+    }
+
+    /**
+     * Retrieve the CSS path relative to the media directory
+     * @param null $store
+     * @return string
+     */
+    public function getCssPath($store = null)
+    {
+        return sprintf(self::CSS_PATH, $this->_getDesignPackage($store), $this->_getDesignTheme($store));
+    }
+
+    /**
+     * Get full URL for CSS file
+     * @param null $store
+     * @return string
+     */
+    public function getCssFileUrl($store = null)
+    {
+        return Mage::getUrl(sprintf('media/%s', $this->getCssPath($store))) . $this->getCssFilename($store);
```

Comment:
I implemented this like:

    /**
     * Generate the URL to access the stylesheet for the store.
     *
     * @param null $store
     * @return string
     */
    public function getCssFileUrl($store = null)
    {
        $url_parts = [
            Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_MEDIA),
            $this->getCssPath($store),
            $this->getCssFilename($store)
        ];

        array_walk($url_parts, function (&$item) {
            $item = trim($item, '/');
        });

        $url = join('/', $url_parts);

        return $this->stripProtocolFromUrl($url);
    }